src/components/fields/location-field/location-helper.ts

```diff
@@ -103,41 +103,72 @@ export namespace LocationHelper {
 		}
 	};
 
+	// Added lat, lng arguments to handle pin location address in query field.
```

Comment:
unnecessary comment, there is no need to explain what has been changed

Reply:
_may I know what are the scenarios where the latlng is different from the query? just trying to understand how are we to deal with such scenarios._

If the user attempts to input/alter their own lat, lng values and "Pin location" text in the search field, then there is a possibility for it to be different from the lat, lng values introduced into the formValues.

Reply:
oh is the user able to type in a value into the input? afaik it needs to be set via code

Reply:
I meant the string value for searching location. I did not want the users to circumvent the search by typing their own "Pin location: lat, lng", that should produce no results.

Reply:
hmm I thought the user is not able to type in their own Pin location: lat, lng in the first place, that's why was thinking we don't even need to prevent the scenario bcos it doesn't happen.

But I may be wrong here. the user can type ?

Reply:
hmm if I try something like this it won't search for me. tested on dev env
![image](https://github.com/LifeSG/web-frontend-engine/assets/3927280/4c99d12f-edfc-4ebb-8261-8863c7a45101)

Reply:
clarified this over code review.

1. The above screenshot is a result of the original bug that this PR is trying to fix
2. We will let the user search for a lat lng using `Pin location: <lat>, <lng>` as long as it's within the map bounds.

@weili-govtech what do you think about point 2?

Reply:
I am fine with point 2 because it is cleaner and behaves more consistently. 
however there is a need to confirm with BA and UX because OS does not behave the same, user cannot search in this way.